packages/node_modules/@webex/common/src/patterns.js

```diff
@@ -11,11 +11,22 @@
  */
 export default {
   /**
-   * Matches an email address by requiring an @ and excluding spaces
-   * @todo add a better email address matcher
+   * Used to validate a string is explicitly an email and allows for immediate
+   * validation of emails provided by services, such as conversations. See
+   * [this link]{@link https://ihateregex.io/expr/email-2} for additional info.
+   *
    * @type {RegExp}
    */
-  email: /^[^\s]+?@[^\s]+?$/,
+  email: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+
+  /**
+   * Used to validate that a ambiguous string contains any emails, and allows
+   * the replacement of all emails within a string. See
+   * [this link]{@link https://ihateregex.io/expr/email-2} for additional info.
+   *
+   * @type {RegExp}
+   */
+  containsEmails: /(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))/,
```

Comment:
Added this RegEx. Note, it functions very differently in contrast to `email` RegEx.
This one:
- Validates if a string contains an email at any character in a string
- Is consumed by the logger plugin
  - Can be used to replace all instances of an email in a string with another string
  - Will not validate if a string is an email